runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -297,6 +306,41 @@ private void playerSkillLookup(ChatMessageType type, SetMessage setMessage, Stri
 		}
 	}
 
+	/**
+	 * Calculates a simple equation and changes the original message to the
+	 * response.
+	 *
+	 * @param messageNode The chat message containing the command.
+	 * @param eqn         The equation to calculate.
+	 */
+	private void calculate(MessageNode messageNode, String eqn)
+	{
+		String result;
+		ChatMessageBuilder builder = new ChatMessageBuilder();
+		try
+		{
+			Expression expr = new ExpressionBuilder(eqn).build();
+			result = String.valueOf(expr.evaluate());
+			builder.append(ChatColorType.NORMAL)
+				.append("Calc: ")
+				.append(eqn)
+				.append(" = ")
+				.append(ChatColorType.HIGHLIGHT)
+				.append(result);
+		}
+		catch (Exception ex)
+		{
+			result = "Calc: Unable to calculate expression: " + eqn;
+			builder.append(ChatColorType.HIGHLIGHT)
+				.append(result);
+		}
+		String response = builder.build();
```

Comment:
Empty line above. Also make response final

Reply:
> Empty line above.

Did you mean add an empty line (if so, I added one for line 337 just now). I don't see an empty line above.

> Also make response final
Done.